monorepo-migration/generate-replace-message-expressions.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+import json
+import logging
+from pathlib import Path
+
+
+log = logging.getLogger(__name__)
+logging.basicConfig(
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    level=logging.DEBUG,
+)
+
+DATA_DIR = (Path(__file__).parent / "data").absolute()
+GITHUB_URL_TEMPLATE = "https://github.com/{repo_owner}/{repo_name}/{number_type}"
+REPO_OWNER = "mozilla-mobile"
+REPO_NAME_TO_IMPORT = "focus-ios"
```

Comment:
Looks like this should be firefox-ios instead... https://github.com/mozilla-mobile/firefox-android/blob/94f8a7e8cc5a18fc7a1c9d10e647fd15a79ac952/monorepo-migration/generate-replace-message-expressions.py#L17C1-L17C20

Reply:
I think changing this line will fix: 

> It seems the prefix `[focus-ios]` wasn't added to each commit message. That's the only issue I see so far.

Let me know if it doesn't

Reply:
Actually, when I change this to firefox-ios, the commit history for focus does not work, the commits are linked to old firefox-ios PRs... not sure why yet...

Reply:
Oh you're right! On Android, it was changed to `firefox-android` for the sake of the migration from the `firefox-android` repo to `mozilla-central` https://github.com/mozilla-mobile/firefox-android/commit/fb86715761d0f3de99c78076612c860687e9faee

Thus, we should stick to `focus-ios`.

Regarding the prefix of the commit message, these lines should do the trick: 

https://github.com/mozilla-mobile/firefox-android/commit/fb86715761d0f3de99c78076612c860687e9faee#diff-99a51aa3f36b7e3ffdd96456612906d550331ee10c456d7d7868135cef17e46fL38-L40